src/base.py

```diff
@@ -13,20 +16,19 @@
 digits[15] = 'F'
 
 
-def change_to_base(n: int, b: int) -> str:
-    """
-    Return `n` in base `b`.
+def change_to_base(i,b):
+        if i== 0:
+            return [0]
+        else:
+            number_in_b=[]
+            while i>0:
+                q=(i//b)
+                r=i%b
+                number_in_b.append(r)
+                i=q
```

Comment:
How does this work if you need to change base on a negative number? The `while i>0` looks like there could be a problem.